frame/dapps-staking/src/lib.rs

```diff
@@ -796,12 +774,120 @@ impl<T: Trait> Module<T> {
                 }),
         }
     }
+
+    fn propagate_total_stakes(stash: &T::AccountId, src_era: &EraIndex, dst_era: &EraIndex) {
+        if <ErasStakingPoints<T>>::contains_key(src_era, stash) {
```

Comment:
`EraStakingPoints` is `(era, **contract_account_id**) => EraStakingPoints`.
I'd like this to work when the user who called the claim is an operator. For some reason, the smart contract itself does not beat the claim.

It might be better to separate the `claim` into `claim_for_nominator` and `claim_for_operator`.

Reply:
modified it in 6816e80